packages/esm-patient-registration-app/src/patient-registration/form-manager.ts

```diff
@@ -131,14 +131,15 @@ export class FormManager {
 
       await this.saveObservations(values.obs, savePatientResponse, currentLocation, currentUser, config);
 
-      const { patientPhotoUuid } = await getConfig('@openmrs/esm-styleguide');
-      if (patientPhotoUuid && capturePhotoProps?.imageData) {
+      const { patientPhotoConceptUuid } = await getConfig('@openmrs/esm-styleguide');
```

Comment:
```suggestion
      const { patientPhotoConceptUuid } = await getConfig<StyleguideConfigObject>('@openmrs/esm-styleguide');
```

Reply:
You'll probably need to add the import anyway

Reply:
Doh 

![CleanShot 2024-06-03 at 6  25 00@2x](https://github.com/openmrs/openmrs-esm-patient-management/assets/8509731/89f16206-ebad-4b70-93c4-bca608bde6fe)

Reply:
Merged that